src/applications/personalization/dashboard/tests/e2e/dashboard-e2e-helpers.js

```diff
@@ -6,18 +6,6 @@ export function mockLocalStorage() {
   );
 }
 
-export const enrollmentStatusEnrolled = {
```

Comment:
No longer needed. Where these were being used before we are now using shared ESR mock responses.